src/host/ut_host/UnicodeLiteral.hpp

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-#define REMOTE_STRING L"�remote �npipe:pipe=foo,server=bar�\t"
+#define REMOTE_STRING L"–remote “npipe:pipe=foo,server=bar”\t"

```

Comment:
Are these even the right type of quotes? Especially since on the next line, they expect 'normal' quotes?